conf/distro/pelux.conf

```diff
@@ -42,3 +42,6 @@ DISTRO_FEATURES_append = " systemd"
 DISTRO_FEATURES_BACKFILL_CONSIDERED += "sysvinit"
 VIRTUAL-RUNTIME_init_manager = "systemd"
 VIRTUAL-RUNTIME_initscripts = "systemd-compat-units"
+
+# Fix for rpm metadata clash between nativesdk-cmake and nativesdk-qtbase-tools
+DIRFILES_pn-nativesdk-cmake = "1"
```

Comment:
Is it possible to put this somewhere other than the distro config? Such as the image bbclass, or local.conf? If not I guess this works, but it doesn't really fit in.

Reply:
As said in the description local.conf works. Did not try the bbclass could try that if you think it is better.

Reply:
bbclass does not work. Shall I move it to `common/local.conf`?

Reply:
If it works to have it in the bbclass, that would make more sense imho.

Reply:
Yes please!

Reply:
fixed